rnd/autogpt_server/autogpt_server/blocks/__init__.py

```diff
@@ -15,6 +16,11 @@
     if os.path.isfile(f) and f.endswith(".py") and not f.endswith("__init__.py")
 ]
 for module in modules:
+    if not re.match("^[a-z_]+$", module):
+        raise ValueError(
```

Comment:
Idk if we should raise or warn for this.

Reply:
From previous experience, it's always safe to just fail on test/app rather than silently allowing stuff.